### DISCUSSION TITLE
Fermata 2.0

### DISCUSSION THREAD

**Discussion by Treenowskyyy** [AndreyPavlenko/Fermata#494] [General]
Hi,
Is there any chance to new update for fermata auto pretty soon ? I mean a new app compatible with Android 14, FS Mirror is working well after DHU glitch but still Its not the same afterall

**Comment by AndreyPavlenko** [comment]
The current version is compatible with A14. The problem is that AA does not officially support such apps. Before A14 there was a workaround, that does not work since A14.

**Comment by rsgorrie** [comment]
@AndreyPavlenko @JUL1EN094 

Hello Andrey I need ur help for something i am working on for my tv station. if u can reach out to me via email [rsgorrie@gmail.com]

thnx u
